docs/cluster-management/ec2-instances.md

```diff
@@ -0,0 +1,59 @@
+# EC2 instances
+
+_WARNING: you are on the master branch, please refer to the docs on the branch that matches your `cortex version`_
+
+You can spin up a Cortex cluster on a variety of AWS instance types. If you are unsure about which instance to pick, review these options as a starting point. This is not a comprehensive guide so please refer to the [full documentation](https://aws.amazon.com/ec2/instance-types/) on AWS for more information.
+
+## T3 instances
```

Comment:
Instead of listing instance families and then listing where it can be used, perhaps list the use cases first and then list out instances that suit the use cases.